src/devPage.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react'
```

Comment:
This can be removed. We're using `dev/devPage.jsx` so this isn't confused with component source code.